akka-actor-typed-tests/src/test/resources/logback-test.xml

```diff
@@ -4,9 +4,6 @@
     <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
-            <level>INFO</level>
-        </filter>
```

Comment:
Not sure why this was set to info, we have the log capturing in place and debug is nice if something fails?